### PR TITLE
Fix broken links to the attribute info in <input type=radio> doc

### PR DESCRIPTION
**Pull request by Schweinepriester**
> What was wrong/why is this fix needed? (quick summary only)

Links on that page pointing to <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#checked> not working.

> MDN URL of main page changed

<https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio>

> Issue number (if there is an associated issue)

No issue (at least not by me)

> Anything else that could help us review it

Not sure if my changes are the correct fix, since `anch()` appears to fill the content as well, but the links are not working since there is no `id="checked"`.

### CODE REVIEW

**Comment by Schweinepriester** [comment]
Alright, <https://pr4084.content.dev.mdn.mozit.cloud/en-US/docs/Web/HTML/Element/input/radio> shows that my changes are not the way to go.

Leaving open to create awareness.
Feel free to close or instruct on how to fix properly or advise to just create an issue for now.

File: files/en-us/web/html/element/input/radio/index.html [hunk 1]
```diff
@@ -49,11 +49,11 @@
   </tr>
   <tr>
    <td><strong>Supported common attributes</strong></td>
-   <td><code>{{anch("checked")}}</code>, <code>{{anch("value")}}</code> and <code><a href="/en-US/docs/Web/HTML/Attributes/required">required</a></code></td>
+   <td><code>{{anch("htmlattrdefchecked")}}</code>, <code>{{anch("value")}}</code> and <code><a href="/en-US/docs/Web/HTML/Attributes/required">required</a></code></td>
```
Comment (rachelandrew) [inline files/en-us/web/html/element/input/radio/index.html:52]:
```suggestion
   <td><code>{{anch("htmlattrdefchecked","checked")}}</code>, <code>{{anch("value")}}</code> and <code><a href="/en-US/docs/Web/HTML/Attributes/required">required</a></code></td>
```

Comment (rachelandrew) [reply files/en-us/web/html/element/input/radio/index.html:52]: I think this should work, adding the display name after the correct link destination.

**Comment by Schweinepriester** [comment]
I had hoped the bot would create a new preview to be able to check... but I'll change the other occurrences as well.

Thanks for the feedback :)

**Comment by sideshowbarker** [comment]
@Schweinepriester Per https://github.com/mdn/content/pull/4084#issuecomment-820414005 this PR seems to be waiting on further changes from you.

**Comment by Schweinepriester** [comment]
@sideshowbarker I did change all occurrences as suggested :) So I think this is waiting for the next review. It would also be great to get another preview like the bot generated to check, if that's possible…

**Comment by sideshowbarker** [comment]
> It would also be great to get another preview like the bot generated to check, if that's possible…

@peterbe could tell us if that’s possible

**Comment by peterbe** [comment]
> I had hoped the bot would create a new preview to be able to check... but I'll change the other occurrences as well.

When you commit more into the PR, it should create a fresh new preview. But clearly, this seems to be broken here :(

Only yesterday, did I fix a bug related to preview upload. Perhaps your PR got stuck in that too. It was related to links that didn't have full `href` value. 
Would you mind merging in the latest upstream `main` and pushing that into this branch to test if it builds again. The bot comment should get a last line about it being edited.

**Comment by Schweinepriester** [comment]
> Would you mind merging in the latest upstream `main` and pushing that into this branch to test if it builds again. The bot comment should get a last line about it being edited.

Done :)

**Comment by Schweinepriester** [comment]
Now there is a [new preview](https://pr4084.content.dev.mdn.mozit.cloud/en-US/docs/Web/HTML/Element/input/radio) (the bot edited the comment), I believe thanks to @sideshowbarker :)

What I've changed looks like it's working as intended.

However there are now flaws being reported:

> -   **h3 heading contains an <a> tag**\
    **HTML:** `<h3 id="htmlattrdefchecked"><a id="attr-checked" href="#attr-checked"><b><code>checked</code></b></a></h3>`\
    *All `<a>` tags need to be removed*
> -   **h3 heading contains an <a> tag**\
    **HTML:** `<h3 id="htmlattrdefvalue"><a id="attr-value" href="#attr-value"><b><code>value</code></b></a></h3>`\
    *All `<a>` tags need to be removed*

However I didn't actively change that and the [live version](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio) is the same, somehow this double `<a>`:

```html
<h3 id="htmlattrdefchecked"><a href="#htmlattrdefchecked" title="Permalink to checked"><a id="attr-checked" href="#attr-checked"><b><code>checked</code></b></a></a></h3>
```

Not sure if want or should address that in this PR?

**Comment by Schweinepriester** [comment]
Thanks again @sideshowbarker, no flaws again :)

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Kai, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Pull request merged**